mpc-recovery/src/oauth.rs

```diff
@@ -65,51 +90,150 @@ pub struct TestTokenVerifier {}
 
 #[async_trait::async_trait]
 impl OAuthTokenVerifier for TestTokenVerifier {
-    async fn verify_token(token: &str) -> Option<&str> {
+    async fn verify_token(token: &str) -> Result<String, String> {
         match token {
             "validToken" => {
-                tracing::info!("TestTokenVerifier: access token is valid");
-                Some("testAccountId")
+                tracing::info!("target: test-token-verifier, access token is valid");
```

Comment:
target value should be outside of quotes 😅 

```suggestion
                tracing::info!(target: "test-token-verifier", "access token is valid");
```

Reply:
Hah, got it :)